app/views/teachers/classrooms/quizzes/manage_students.html.slim

```diff
@@ -0,0 +1,30 @@
+- content_for :classroom_show_content do
+  h1 = @quiz.name
+  .spacer-small
+
+  .row
+    .large-10.large-push-1.columns
+      table#students-table.display
+        thead
+          tr
+            th Email
+            th Quiz Completed
+            th Actions
+        tbody
+          - @classroom.students.each do |student|
+            tr id="#{dom_id student}"
+              td = student.email
+              td
+                - if student.send(@quiz.name.split.first.downcase + '_quiz_completed') < 0
+                  = "No"
+                - else
+                  = student.send(@quiz.name.split.first.downcase + '_quiz_completed')
```

Comment:
Looks like you're doing this a lot. Usually we put these methods into a Decorator (https://github.com/drapergem/draper) to make the view cleaner. In general, we want to minimize the amount of variables set in the controller and make the view as simple as we can.

This means you want to create a decorator called `quiz_decorator.rb` and then make a method that does this, say called `student_column_name`.

Then you can just do `student.send(@quiz.student_column_name)` in the view.

It also looks like you do this computation in `edit_quiz_score` in the students controller. Instead of making a variable containing, you can just use your decorator method.

Let me know if you have any questions about this. Everything should work the same as before but the code will be cleaner.